stable/unifi/templates/pvc.yaml

```diff
@@ -21,4 +26,4 @@ spec:
   storageClassName: "{{ .Values.persistence.storageClass }}"
 {{- end }}
 {{- end }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
```

Comment:
```suggestion
```

Reply:
I tested this one and it broke so I left this one as is. This end is for the if statement in line 1